provider/cmd/crd2pulumi/gen/generate.go

```diff
@@ -42,7 +42,7 @@ const (
 )
 
 // Version specifies the crd2pulumi version.
-const Version = "1.0.2"
+var Version = "1.0.2"
```

Comment:
```suggestion
var Version
```

This should always be set by the linker. If it's important to any of the internal logic, you should assert that it's set.